src/components/generics/vacation-test-forms/myVacationMockData.json

```diff
@@ -2,13 +2,13 @@
     {
```

Comment:
Does this mock data actually need to be commited, is it used in some tests? If not then this file should be deleted or added to the .gitignore if you need to use it locally